includes/classes/Product.php

```diff
@@ -99,7 +102,12 @@ public function getId(): ?int
 
     public function exists(): bool
     {
-        return !empty($this->product_id) && !empty($this->data);
+        if (!empty($this->product_id)) { 
+            if (!empty($this->data['products_name']) || $this->has_description) {
+                return true; 
```

Comment:
there is no need for a nested `if` here.

in addition, i am not sure about the logic.  we have a product object which we are now saying it does not exist.  but it does exist, but it is missing a description.  how could one differentiate between a product_id which has no data and one which has data but no product description?  i would think the product object should be able to provide that information as opposed to saying they are the same; which this code does.

handling a product with no description by saying it does not exist just seems wrong to me.  it makes it ambiguous.